src/backend/opencl/cl/rx/randomx.cl

```diff
@@ -8,6 +8,8 @@
 #include "randomx_constants_loki.h"
 #elif (ALGO == ALGO_RX_ARQMA)
 #include "randomx_constants_arqma.h"
+#elif (ALGO == ALGO_RX_V)
+#include "randomx_constants_v.h"
```

Comment:
There's no need to add OpenCL constants for rx/v - they're exactly the same as rx/0, see https://github.com/xmrig/xmrig/pull/1434 for reference

Reply:
Basically, you just need to change algo id in OclRxBaseRunner.cpp from RX_V to RX_0 and that will be enough for OpenCL support. Dataset is generated on CPU with correct Arong2d salt, everything else can just run with rx/0 code.

Reply:
I made required changes.